.changeset/cyan-pears-yawn.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We can skip the changeset here as we only want to remove the deprecations from the `.test.` files and those do not require changesets. The rest of them require a major bump so we want to keep them for now :)

Reply:
Okey! Could you help me with how I should do to reverse the changeset?